Source/track.cpp

```diff
@@ -115,7 +115,8 @@ void RepeatMouseAction()
 			auto &object = Objects[pcursobj];
 			if (object.IsDoor())
 				break;
-			NetSendCmdLocParam1(true, CMD_OPOBJXY, object.position, pcursobj);
+			// This should probably be cursPosition so paths to large objects are consistent
+			NetSendCmdLoc(MyPlayerId, true, CMD_OPOBJXY, object.position);

```

Comment:
Finally worked out what was breaking the timedemo test.

Large objects like sarcophagi can end up with different pathfinding depending on which tile the cursor is over when interacting with them. Using the object position isn't correct as this is always the southernmost tile, the cursor might be over another tile and the player might want to path to a different edge.

However repeated mouse actions break pathing and force navigating to the southernmost tile. This gets triggered twice in the demo, the pathfinding diverges if I try fix this.

Reply:
Nice work! That can't have been easy to track down. I guess who ever fixes this have to also record a new demo.

Reply:
Which is why I'm trying to avoid fixing it 😂

Reply:
At least you added a comment :)